### ISSUE TITLE
Remove `implicit-casts` for new Dart versions or use this workaround

### ISSUE THREAD

**Issue by minhqdao** [passsy/dart-lint#65]
Hellou, 👋🏽

I'm getting the following warning when including `package:lint/strict.yaml` (or the equivalent `casual.yaml` or `package.yaml` file) while I'm on Dart version `3.0.0` (build `3.0.0-218.1.beta`):

```
Warning in the included options file <path>/lib/strict.yaml(227..240): The option 'implicit-casts' is
no longer supported.dart(included_file_warning)
```

It seems that the `implicit-casts` option needs to be removed for Dart versions >= `3`. As I couldn't figure out conditional includes (I don't think they exist for `yaml` files), maybe we need additional files without the option so the user can use them for newer Dart versions.

Anyways, a current workaround would be to override `strong-mode` by making sure your `analysis_options.yaml` file contains:

```yaml
package:lint/package.yaml

analyzer:
  strong-mode:
```

This will remove the warning, too. 😋

**Comment by passsy** [comment]
Thanks for reporting, this has been adjusted in `lint: 2.1.0`
